codespeed/templates/codespeed/timeline.html

```diff
@@ -3,6 +3,7 @@
 {% block title %}{{ block.super }}: Timeline{% endblock %}
 
 {% block extra_head %}
+{{ block.super }}
```

Comment:
Introduced: https://github.com/tobami/codespeed/commit/904a0c3ec40e542d6e1412c810ad44596e50c003
Removed: https://github.com/tobami/codespeed/commit/ec9cd1b6ebca6e031ee1ecbfc2b147f26d5799c6
The removal doesn't have a very clear diff, but it looks like the change was accidental.